salt/metalk8s/salt/minion/files/minion-99-metalk8s.conf.j2

```diff
@@ -14,4 +14,4 @@ saltenv: {{ saltenv }}
 # Prepare volume at startup (required for loop devices persistency).
 startup_states: sls
 sls_list:
-  - /metalk8s/volumes/prepared
+  - /metalk8s/volumes/provisioned
```

Comment:
Applying this state may easily fail (potentially only partially). Is that handled gracefully by Salt?

Reply:
What I'm aiming at: it's possible `provision` fails because `exists` is `False` (e.g., we didn't prepare the sparse file yet).

Reply:
I think the output of the state is just ignored

Reply:
If that's the case (and logs aren't being spammed), then that's OK for me.

Reply:
That doesn't prevent the minion to start.

Just have an error logged:

```
[DEBUG   ] minion return: {u'fun_args': [[u'/metalk8s/volumes/provisioned']], u'jid': u'req', u'return': {u'test_|-Provision backing storage for kotlet_|-Provision backing storage for kotlet_|-fail_without_changes': {u'comment': u'Failure!', u'name': u'Provision backing st
orage for kotlet', u'start_time': '15:07:37.884802', u'result': False, u'duration': 0.948, u'__run_num__': 0, u'__sls__': u'/metalk8s/volumes/provisioned', u'changes': {}, u'__id__': u'Provision backing storage for kotlet'}}, u'retcode': 2, u'success': True, u'fun': u'stat
e.sls'}
```